setup.py

```diff
@@ -12,19 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #!/usr/bin/env python3
+import os
 import re
 from skbuild import setup
 
 
 with open("./pennylane_qrack/_version.py") as f:
     (version,) = re.findall('__version__ = "(.*)"', f.read())
 
+if not os.path.exists("qrack/_build"):
+    os.mkdir("qrack/_build")
+os.chdir("qrack/_build")
+os.system("cmake -DCPP_STD=14 .. ; make all -j(nproc --all)")
+os.chdir("../..")
```

Comment:
It is usually recommended (for OS compatibility to do `os.chdir(os.path.join("..", ".."))` over `os.chdir("../..")`.

Reply:
(Outdated.)